diesel/src/query_builder/insert_statement.rs

```diff
@@ -335,7 +335,7 @@ where
 {
 }
 
-impl<T, Table> UndecoratedInsertRecord<Table> for [T]
```

Comment:
This is also a breaking change

Reply:
I guess technically? We can keep it around, it literally does nothing though